index.d.ts

```diff
@@ -37,6 +37,11 @@ declare namespace init {
       message: string,
       methodName: 'assert' | 'debug' | 'error' | 'info' | 'log' | 'warn'
     ) => boolean
+
+    /**
+     * This parameter lets you define a list of test names to skip console checks for.
+     */
+    skipTestNames?: []
```

Comment:
```suggestion
    skipTestNames?: string[]
```

Reply:
Whoops my bad,  cc2bdc4485ab79273d4d88ac253e3e72f0f6adf3 thanks for the suggestion!